lib/rules/indent.js

```diff
@@ -1011,6 +1011,11 @@ module.exports = {
 
             VariableDeclarator(node) {
                 if (node.init) {
+                    const operator = sourceCode.getTokenBefore(node.init);
+
+                    if (operator.type === "Punctuator" && operator.value === "=" && tokenInfo.isFirstTokenOfLine(operator)) {
+                        offsets.ignoreToken(operator);
```

Comment:
`offsets.ignoreToken` is a no-op if a token is not the first token of its line, so it's not necessary to check for that in the `if` statement.